utils/build.ps1

```diff
@@ -329,6 +326,14 @@ $BuildArch = switch ($BuildArchName) {
   default { throw "Unsupported processor architecture" }
 }
 
+$WiXVersion = "4.0.6"
+$WiX = @{
+  Version = $WiXVersion;
+  URL = "https://www.nuget.org/api/v2/package/wix/$WiXVersion";
```

Comment:
I think that the dynamism is pointless. The SH256 is checked and will fail if the URL changes. Let's follow the more data driven approach and version this thoroughly.